testing/integration/test/bin/load-zork.sh

```diff
@@ -19,6 +19,9 @@ while getopts b:r:lpvh? opt; do
         p)
             PREBUILT=true
             ;;
+        n)
+            NPM=true
```

Comment:
Let's declare this at the top, like we have the others (I know this will work but let's maintain the convention of the rest of the file).